apps/dashboard/src/fairness/utils.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IMetricRequest, IMetricResponse } from "@responsible-ai/core-ui";
+
+export const supportedBinaryClassificationPerformanceKeys = [
+  "accuracy_score",
+  "balanced_accuracy_score",
+  "precision_score",
+  "recall_score",
+  "f1_score"
+];
+
+export const supportedRegressionPerformanceKeys = [
+  "mean_absolute_error",
+  "r2_score",
+  "mean_squared_error",
+  "root_mean_squared_error"
+];
+
+export const supportedProbabilityPerformanceKeys = [
+  "auc",
+  "root_mean_squared_error",
+  "balanced_root_mean_squared_error",
+  "r2_score",
+  "mean_squared_error",
+  "mean_absolute_error"
+];
+
+export const messages = {
+  LocalExpAndTestReq: [{ displayText: "LocalExpAndTestReq" }],
+  LocalOrGlobalAndTestReq: [{ displayText: "LocalOrGlobalAndTestReq" }],
+  PredictorReq: [{ displayText: "PredictorReq" }],
+  TestReq: [{ displayText: "TestReq" }]
+};
+
+// calculateMetrics can be used with local flask server for debugging
+/*export function calculateMetrics(postData: IMetricRequest): Promise<IMetricResponse> {
```

Comment:
remove commented out code?

Reply:
This isn't new in this PR (it just moved files) and actually commented out on purpose. It can be used if one wants to debug with flask using Jupyter. That's why it uses localhost